store/workspace_setting.go

```diff
@@ -140,6 +140,8 @@ func (s *Store) GetWorkspaceGeneralSetting(ctx context.Context) (*storepb.Worksp
 const (
 	// DefaultContentLengthLimit is the default limit of content length in bytes. 8KB.
 	DefaultContentLengthLimit = 8 * 1024
+	// DefaultReaction = thumbs up.
+	DefaultReaction = "\U0001f44d"
```

Comment:
```suggestion
	DefaultReactions = []string{"👍"}
```

Reply:
I've update the code to use the emoji rather than the string representation of the emoji. Strangly when I tried using the emoji before my ide showed a compile error stating something to do with utf8. Cant replicate again so all good.

I tried setting the reactions as an array bit you cant set an array as a constant. If we intend to add more default reactions I'll pull the list into a variable.
![image](https://github.com/user-attachments/assets/16c7d3d8-6f6b-4968-b669-361b893692ab)